### PR TITLE
[RHPAM-1505] RH-SSO and LDAP absent in RHPAM OpenShift immutable server templates

### PR DESCRIPTION
**Pull request by errantepiphany**
[RHPAM-1505] RH-SSO and LDAP absent in some RHPAM OpenShift templates
https://issues.jboss.org/browse/RHPAM-1505

Signed-off-by: David Ward <dward@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by errantepiphany** [comment]
@jakubschwan Oops, you're right. Not sure how I missed that one. It's there now!

**Comment by errantepiphany** [comment]
@jakubschwan @spolti Should be all set now, finally? (LOL) Can I get an approval please?

File: templates/rhpam71-trial-ephemeral.yaml [hunk 1]
```diff
@@ -66,6 +65,11 @@ parameters:
   name: KIE_SERVER_CONTROLLER_USER
   value: controllerUser
   required: false
+- displayName: KIE Server Controller Protocol
+  description: KIE server controller protocol (Used to set the org.kie.server.controller system property)
```
Comment (spolti) [inline templates/rhpam71-trial-ephemeral.yaml]: upper case?

Comment (errantepiphany) [reply templates/rhpam71-trial-ephemeral.yaml]: @spolti in all the other controller parameters, the displayName uses upper case, but the description uses lower case. I was just being consistent...

**Pull request merged**